packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -136,21 +136,22 @@ class InteractionResponses {
   }
 
   /**
-   * Edits the initial reply to this interaction.
+   * Edits a reply to this interaction.
    * @see Webhook#editMessage
    * @param {string|MessagePayload|WebhookEditMessageOptions} options The new options for the message
+   * @param {MessageResolvable|"@original"} [message="@original"] The response to edit
    * @returns {Promise<Message>}
    * @example
-   * // Edit the reply to this interaction
+   * // Edit the initial reply to this interaction
    * interaction.editReply('New content')
    *   .then(console.log)
    *   .catch(console.error);
    */
-  async editReply(options) {
+  async editReply(options, message = '@original') {
```

Comment:
Shouldn't this parameter be merged with `options`?

Reply:
I also think it should be in options